### DISCUSSION TITLE
rasterio merge in geowombat

### DISCUSSION THREAD

**Discussion by jozphlp** [jgrss/geowombat#135] [Q&A]
Hi all, just wondering if there's a way to implement the equivalent of mosaic with rasterio merge in geowombat? Thanks

**Comment by mmann1123** [comment]
Yes please refer to our [read the docs](https://geowombat.readthedocs.io/en/latest/tutorial-open.html#opening-multiple-bands-as-a-mosaic), and for more general spatial programming book see [www.pygis.io](www.pygis.io)

**Comment by jgrss** [comment]
Hi @jozphlp the [plotting examples](https://geowombat.readthedocs.io/en/latest/tutorial-plot.html) might also be a helpful demonstration of mosaicking.
